.templates/blynk_server/service.yml

```diff
@@ -3,8 +3,9 @@
     container_name: blynk_server
     restart: unless-stopped
     ports:
-      - "8180:8080"
-      - "8441:8441"
-      - "9443:9443"
+      - 8180:8080
```

Comment:
Ports must have double quotes around them.

Reply:
Really? I am not sure about this.
In the examples what I have seen there were no double quotes.

Reply:
Yup I realised this after submitting the PR, Grahams ogirinals did not have quotes but I noticed in the code changes you had updated for quotes".